packages/cloudcasa/package.yaml

```diff
@@ -1,2 +1,4 @@
-url: https://github.com/catalogicsoftware/cloudcasa-helmchart/releases/download/cloudcasa-1.0/cloudcasa-1.0.tgz 
+url: https://github.com/catalogicsoftware/cloudcasa-helmchart.git
+subdirectory: v1.0.0
+commit: f22253e08cbacc752f1b6e052d01c87ca3ac1555
```

Comment:
Since we're shifting to pointing to a Git repository, the Chart.yaml flip-flopped a bit. If you look at the actual files of the diffs, the changes are identical.